tests/cli/usage.spec.ts

```diff
@@ -0,0 +1,28 @@
+import { describe, expect, it, jest } from '@jest/globals';
+import { createUsage } from '../../src/cli/usage';
+
+describe('createUsage', () => {
+  it('uses console.log', () => {
+    console.log = jest.fn();
+    createUsage();
+    expect(console.log).toHaveBeenCalledTimes(1);
+  });
+
+  it('returns an usage message', () => {
+    console.log = jest.fn();
+    createUsage();
+
+    const usage = `
+  Usage:
```

Comment:
This doesn't have the wrong indentation, it must be like that in order to match what `createUsage()` returns.